nginx/tripleawarclub.org

```diff
@@ -2,7 +2,7 @@ server {
     listen 80;
     listen [::]:80;
     server_name tripleawarclub.org www.tripleawarclub.org;# dice.tripleawarclub.org *.tripleawarclub.org;
-    rewrite     ^   https://$host$request_uri? permanent;
+    return 301 https://forums.triplea-game.org;
```

Comment:
I actually find this notation far more convenient ^^
I learned that it exists today.